src/pytypes/onefuzztypes/models.py

```diff
@@ -794,6 +794,11 @@ class Task(BaseModel):
     user_info: Optional[UserInfo]
 
 
+class NetworkConfig(BaseModel):
+    address_space: str = Field(default="10.0.0.0/8")
```

Comment:
Maybe we should pass these values via named constants.

Reply:
I thought about that, but didn't want the named constants used elsewhere, rather always just use the defaults from the model directly.